src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Button,
+  createStyles,
+  Grid,
+  Switch,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
+
+import { LayerType, TableType } from '../../../config/types';
+import { addLayer, removeLayer } from '../../../context/mapStateSlice';
+import { loadTable } from '../../../context/tableStateSlice';
+import { layersSelector } from '../../../context/mapStateSlice/selectors';
+import { menuList } from '../utils';
+
+function MenuItemMobile({ classes }: MenuItemMobileProps) {
+  const [expanded, setExpanded] = useState<string>('');
```

Comment:
the `<string>` type assertion isn't necessary, since it can be inferred with `''`